policy_packs/aws/stack/deploy_aws_iam_stack/policies.tf

```diff
@@ -0,0 +1,73 @@
+# AWS > IAM > Stack [Native]
+resource "turbot_policy_setting" "aws_iam_stack" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStackNative"
+  value    = "Check: Configured"
+  # value    = "Enforce: Configured"
+}
+
+# AWS > IAM > Stack [Native] > Variables
+resource "turbot_policy_setting" "aws_iam_stack_variables" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStackNativeVariables"
+  value    = file("./stack/variables.auto.tfvars")
```

Comment:
The file path would need changes if are moving these files within the Policy Pack directory.

Reply:
See https://github.com/turbot/guardrails-samples/pull/912#discussion_r1929063117 ...